geospaas_harvesting/harvest.py

```diff
@@ -209,7 +210,13 @@ def main():
     except AssertionError:
         LOGGER.error('Invalid configuration', exc_info=True)
         sys.exit(1)
+    LOGGER.info('Executing update_vocabularies command before commencing the harvesting process in order to keep the vocabularies updated.')
+    #subprocess.run(["python", "settings.configure()"])  #subprocess.call()
+    #subprocess.run(["chmod", "+x", "manage.py"])  #subprocess.call()
+    update_vocabularies.Command().handle() #updating the vocabulary with this command
 
+    #subprocess.run(["python", "manage.py", "update_vocabularies"])  #subprocess.call()
+    LOGGER.info('updating the vocabularies is accomplished, now start the harvesting process')
```

Comment:
Try to keep the log messages short and simple, and please remove commented code

```suggestion
    LOGGER.info('Updating vocabularies...')
    update_vocabularies.Command().handle()
    LOGGER.info('Finished updating vocabularies')
```

Reply:
Done in the next commit